source/scripts/model/Monster.js

```diff
@@ -100,6 +100,28 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit() {
+          //Variables for the monster and hero positions
+       var monX = this.position.x
+       var monY = this.position.y
+       var adX = this.game.adventurer.position.x 
+       var adY = this.game.adventurer.position.y
```

Comment:
When I said I "really dislike this alias-ing here", you'll note I was talking about the variables that were just reassigning the same value to another variable to make it a biiiit shorter, namely the `monX`, `monY`, `adX`, and `adY`.

What I wasn't talking about was `dx` and `dy`, which actually perform a meaningful calculation, and is fine to save in a new variable.